scripts/deploy.js

```diff
@@ -16,6 +16,15 @@ const spawnWithErrorHandling = (...args) => {
   }
 };
 
+let tagname = 'latest';
+if (version.includes('alpha')) {
```

Comment:
Good idea! Updated the PR. Will fallback to `pre` if match fails. Relies on pre-release format to be: `<version>-<tag>.<#>`